ui/app/templates/components/calendar-widget.hbs

```diff
@@ -59,46 +57,72 @@
     {{#if this.showCalendar}}
       <div class="calendar-widget-container" data-test-calendar-widget-container>
         <div class="select-year">
-          <Hds::Button
-            data-test-previous-year
-            id="previous-year"
-            disabled={{this.disablePastYear}}
-            {{on "click" this.subYear}}
-            @color="secondary"
-            @text="previous year"
-            @icon="chevron-left"
-            @isIconOnly={{true}}
-          />
-          <p data-test-display-year>
+          {{#if this.disablePastYear}}
+            <Hds::TooltipButton
+              @text={{concat
+                (sub this.displayYear 1)
+                " is unavailable because it is before your start date. Change your start month to a date in "
+                (sub this.displayYear 1)
+                " to see data for this year."
+              }}
+              aria-label="More information about why previous year is disabled"
+            >
+              <Hds::Button
+                data-test-previous-year
+                id="previous-year"
+                disabled={{true}}
+                {{on "click" this.subYear}}
+                @color="secondary"
+                @text="previous year"
+                @icon="chevron-left"
+                @isIconOnly={{true}}
+              />
+            </Hds::TooltipButton>
+          {{else}}
+            <Hds::Button
+              data-test-previous-year
+              id="previous-year"
+              {{on "click" this.subYear}}
+              @color="secondary"
+              @text="previous year"
+              @icon="chevron-left"
+              @isIconOnly={{true}}
+            />
+          {{/if}}
+
+          <Hds::Text::Display data-test-display-year>
             {{this.displayYear}}
-          </p>
-          <Hds::Button
-            data-test-next-year
-            id="next-year"
-            disabled={{this.disableFutureYear}}
-            {{on "click" this.addYear}}
-            @color="secondary"
-            @text="next year"
-            @icon="chevron-right"
-            @isIconOnly={{true}}
-          />
-          {{#if this.tooltipTarget}}
-            {{! Component must be in curly bracket notation }}
-            {{! template-lint-disable no-curly-component-invocation }}
-            {{#modal-dialog
-              tagName="div"
-              tetherTarget=this.tooltipTarget
-              targetAttachment="top right"
-              attachment="top middle"
-              offset="150px 0"
-            }}
-              <div class={{"calendar-tooltip"}}>
-                <p>
-                  {{this.tooltipText}}
-                </p>
-              </div>
-              <div class="chart-tooltip-arrow"></div>
-            {{/modal-dialog}}
+          </Hds::Text::Display>
+
+          {{#if this.disableFutureYear}}
+            <Hds::TooltipButton
```

Comment:
warning: this is a "nested interactive" [a11y violation, @MelSumner can confirm]

if you want to apply a tooltip to an interactive element like a button, you have to use the specific `{{hds-tooltip}}` modifier https://helios.hashicorp.design/components/tooltip?tab=code#ember-modifier

Reply:
I'm aware -- this PR is not going to fix all of the issues, and we plan to swap this component out with the HDS dropdown in a different PR